### DISCUSSION TITLE
Config.toml

### DISCUSSION THREAD

**Discussion by OmarSa90** [whitphx/stlite#655] [General]
Hello, has anyone succeeded in loading config.toml within the stlite?

to be worked on the multi pages

thank you in advanced

**Comment by whitphx** [comment]
Since 0.40.0, you can use the [`streamlitConfig` option](https://github.com/whitphx/stlite?tab=readme-ov-file#customizing-the-streamlit-configuration-streamlitconfig-option) in the case of `@stlite/mountable`.

Since 0.62.1, Stlite supports parsing `.streamlit/config.toml`:
https://edit.share.stlite.net/#!ChBzdHJlYW1saXRfYXBwLnB5EkcKEHN0cmVhbWxpdF9hcHAucHkSMwoxaW1wb3J0IHN0cmVhbWxpdCBhcyBzdAoKc3QudGl0bGUoIkhlbGxvIFdvcmxkISIpChJPChYuc3RyZWFtbGl0L2NvbmZpZy50b21sEjUKM1t0aGVtZV0KYmFja2dyb3VuZENvbG9yID0gIndoaXRlIgp0ZXh0Q29sb3IgPSAicmVkIg,=

Thanks
